cropper/src/main/kotlin/com/canhub/cropper/BitmapUtils.kt

```diff
@@ -457,9 +457,13 @@ internal object BitmapUtils {
   ): Uri {
     val newUri = customOutputUri ?: buildUri(context, compressFormat)
 
-    return context.contentResolver.openOutputStream(newUri, WRITE_AND_TRUNCATE).use {
-      bitmap.compress(compressFormat, compressQuality, it)
-      newUri
+    return try {
+      context.contentResolver.openOutputStream(newUri, WRITE_AND_TRUNCATE).use {
+        bitmap.compress(compressFormat, compressQuality, it)
+        newUri
+      }
+    } catch (e: SecurityException) {
+      throw SecurityException(e)
```

Comment:
What are you trying to achieve here? I don't understand it

Reply:
We are trying to solve the issue #587

Reply:
You are catching the error and then throwing it, how is that solving anything?